### ISSUE TITLE
po-mode for Emacs is not installed with gettext

### ISSUE THREAD

**Issue by michalrus** [NixOS/nixpkgs#17466]
## Issue description

GNU gettext contains `.el` files defining the `po-mode` used to edit `.po` files in Emacs. These are not installed to `share/emacs/site-lisp`.

Quick fix: https://github.com/michalrus/dotfiles/commit/9eec8965715e95e72c77d4dfe34a30ed3fa5f59b
### Steps to reproduce
1. Add gettext to system packages.
2. Run emacs.
3. Try loading `po-mode`. It doesn’t exist.
## Technical details
- System: 16.09pre87839.99f63b4 (Flounder)

**Comment by Profpatsch** [comment]
(triage) was this fixed by now?

**Comment by michalrus** [comment]
I don’t know

**Comment by Profpatsch** [comment]
tbh I’m not quite sure what the original problem is. What part of nixpkgs is missing gettext? The emacs closure? Some module? Or should gettext provide the emacs mode?

**Comment by michalrus** [comment]
> Or should gettext provide the emacs mode?

Yes, that last option. `gettext` package has some `*.el` files, but they’re not visible to Emacs, IIRC.

**Comment by Profpatsch** [comment]
Hm, `nix-build -A gettext.all && (find result*/ | grep "\.el")` doesn’t find any elisp files here. Maybe the gettext build doesn’t generate them?

**Comment by Profpatsch** [comment]
@michalrus Ah, then it shouldn’t be hard to put that into nixpkgs, right? Maybe change the kind of hacky `find` to a series of explicit `mv` operations, but other that that it looks sensible. You could also think about putting it into its own output, like `emacs` or `elisp` (I don’t know what’s the convention here, maybe grep your store for it).

**Comment by qweered** [comment]
I see https://github.com/michalrus/dotfiles/blob/d32a4657505f3adb3ab5f97a8b8296076dc602b2/packages/gettext-emacs/default.nix still exists so assuming this is not fixed
